### PR TITLE
Removed one role

### PR DESCRIPTION
**Pull request by lesioTheProgrammer**

### CODE REVIEW

File: RoomMate.Api/Controllers/DashboardController.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ public IActionResult AddHouseWork([FromBody]HouseWorkDto houseWorkDto)
             }
 
             return this.BadRequest();
+            //
```
Comment (Dhakkain) [inline RoomMate.Api/Controllers/DashboardController.cs]: Remove

File: RoomMate.Api/Controllers/UserManagementController.cs [hunk 2]
```diff
@@ -25,8 +25,23 @@ public class UserManagementController : Controller
             this.tokenService = tokenService;
             this.userService = userService;
         }
+        //cities with address at once?
+        //NOOOOOOOOOOOOOO
```
Comment (Dhakkain) [inline RoomMate.Api/Controllers/UserManagementController.cs]: ?

File: RoomMate.Api/Controllers/UserManagementController.cs [hunk 3]
```diff
@@ -25,8 +25,23 @@ public class UserManagementController : Controller
...
+        [Route("GetCities")]
+        [HttpGet]
+        public IActionResult GetCitiesByTwoLett(string letters)
+        {
+            return this.Ok(this.registerService.GetCitiesByTwoLett(letters));
```
Comment (Dhakkain) [inline RoomMate.Api/Controllers/UserManagementController.cs]: GetCitiesByName *

File: RoomMate.Api/Controllers/UserManagementController.cs [hunk 4]
```diff
@@ -25,8 +25,23 @@ public class UserManagementController : Controller
...
+        }
+
+
+        [Route("GetAddress")]
+        [HttpGet]
+        public IActionResult GetAddrByCityID(int id, string streetLetters)
```
Comment (Dhakkain) [inline RoomMate.Api/Controllers/UserManagementController.cs]: Shortcut Addr?

Comment (Dhakkain) [reply RoomMate.Api/Controllers/UserManagementController.cs]: ID = Id

File: RoomMate.Domain/Dto/AddressDto.cs [hunk 5]
```diff
@@ -1,6 +1,15 @@
 ﻿namespace RoomMate.Domain.Dto
 {
-    public class AddressDto
+    public class AddressDto : BaseDto
     {
+        public int CityId { get; set; }
+        public string CityName { get; set; }
+        public int AddressID { get; set; }
```
Comment (Dhakkain) [inline RoomMate.Domain/Dto/AddressDto.cs]: Id

File: RoomMate.Front/RoomMatePortal/src/app/address/dto/address-dto.ts [hunk 6]
```diff
@@ -0,0 +1,8 @@
+
+ export class AddressDto{
+  public cityid: number;
```
Comment (Dhakkain) [inline RoomMate.Front/RoomMatePortal/src/app/address/dto/address-dto.ts]: cityId - name should be the same api = front (first letter small)

File: RoomMate.Front/RoomMatePortal/src/app/address/dto/city-dto.ts [hunk 7]
```diff
@@ -0,0 +1,4 @@
+export class CityDto {
+  public cityid: number;
```
Comment (Dhakkain) [inline RoomMate.Front/RoomMatePortal/src/app/address/dto/city-dto.ts]: same like upper com

**Comment by lesioTheProgrammer** [comment]
Fixed

**Pull request closed without merging**